app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,14 @@
+module SessionsHelper
+
+  def log_in user
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by id: session[:user_id]
+  end
+
+  def logged_in?
```

Comment:
Bạn xem lại phần `Controller filters and helpers` do `gem devise` cung cấp để sử dụng các method trong helper này nhé